src/en/components/radio-group/base.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: 'layouts/base.njk'
+github: https://github.com/cds-snc/gcds-components/tree/main/packages/web/src/components/gcds-radio-group
+figma: https://www.figma.com/file/mh2maMG2NBtk41k1O1UGHV/Canadian-Digital-Service%E2%80%A8---GC-Design-System?node-id=818%3A3759&t=ciEmm7GYyGAY73zZ-0
+permalink: false
+tags: ['radiogroupEN', 'header']
+---
+
+# Radio Group <br>`<gcds-radio-group>`
+
+_Also called: radio button, radio._
+
+The radio group is a set of options for a single selection.
+
+{% docLinks locale stage figma github %}
+{% enddocLinks %}
+
+{% componentPreview "Radio group component preview" "px-300 pt-400 pb-200" %}
+<gcds-fieldset fieldset-id="fieldset" legend="Legend" hint="Hint / Example message.">
+<gcds-group-radio name="radio" options="[{'id':'form-radio-1', 'label': 'Label 1', 'hint':'Description or example to make the option clearer'},{'id':'form-radio-r', 'label': 'Label 2', 'hint':'Description or example to make the option clearer' }]">
```

Comment:
The radio group doesn't get loaded properly in the preview example:

![Screenshot 2024-04-15 at 9 16 37 AM](https://github.com/cds-snc/gcds-docs/assets/8675814/52e4e3b2-991f-4aa1-b6b1-db1e48a55d55)

Reply:
It won't load until we've merged https://github.com/cds-snc/gcds-components/pull/486
So it needs to be published right after we merge that one in.

At the moment we only have Radio published: 
https://cds-snc.github.io/gcds-components/?path=/docs/components-radio--overview

So as soon as that one's merged, our docs site showing the radio page will break too!

Reply:
Right, totally spaced and forgot that we haven't published that component yet :D